### ISSUE TITLE
Could Not Reach Host

### ISSUE THREAD

**Issue by amuttsch** [clementine-player/Android-Remote#2]
_From [sg2chibi...@gmail.com](https://code.google.com/u/115200288686560071837/) on March 02, 2013 21:29:24_

What steps will reproduce the problem? 1. Entering my IP and attempting to connect What is the expected output? What do you see instead? Expected: the remote to connect. Instead: a message that it cannot reach host. What version of the product are you using? On what operating system? Latest Clementine build, 32-bit, Fedora 18. App downloaded from Google Play. Please provide any additional information below. I've gone and updated my version of Clementine from the stable to latest (lots of nice new features), got the app installed on my phone, the remote is enabled and the port is 5500 on both my computer as well as in the app's settings...but when I enter my IP address ( http://www.whatismyip.com/ ), the app tells me it cannot connect to the host. My computer is on and playing music at this very moment, but the app won't connect to it. Not sure what I'm doing wrong.

_Original issue: http://code.google.com/p/clementine-remote-android/issues/detail?id=2_

**Comment by Ketwo** [comment]
Hey there,
I also seem to have problems connecting.
Using Clementine 1.3.1 on Windows 10 Enterprise N  and remote 11.1 on a Nexus 5 with Android 6.0.1
No proxy settings that I know off, port 5500 and using the ipaddress that the clementine preferences menu shows (192.168.0.108).
I turned the windows firewall off but got the same error. I tried setting an incoming rule in windows firewall on port 5500, still the error.
I ran netstat -a and did not see port 5500 listening on the given ip address, but it did show the following:
TCP [::]:5500 Desktop-etc:0 LISTENING, so it seems to me that the pc is listening.
Hope someone can help me. Thanks!

**Comment by j3009** [comment]
Hi Ketwo,
Did you solved your problem? 
I heve the same.
Thanks

**Comment by Ketwo** [comment]
Unfortunatly, no luck yet.

**Comment by MJae** [comment]
App worked like a couple of times and never did again. All questions are clear in the basic troubleshooter and it's still no good. I'm hoping for an update, but since the last one is from September 12, 2016 (according to the Google Play Store), I guess that's not happening.

**Comment by tambux** [comment]
Debian GNU/Linux 10.4 here, Clementine 1.3.1+git609-g623a53681+dfsg-1 from the Debian Repository.

For Me, **netstat -a** is showing that Clementine is listening, but only in IPv6 port
tcp6       0      0 [::]:5500               [::]:*                  LISTEN

However, in Clementine Preferences, the Private Networks/IP Addresses listed are IPv4 Addresses. When I look to my Firewall Logs, I can see Clementine Remote trying to access TCP4/5500. My Wifi does not support IPv6, 

Basically, the Problem in My Setup is that Clementine listens only IPv6, and Android is only able to connect IPv4. The only solution I came up with, is to redirect TCP4/5500 to TCP6/5500 like this:
**socat TCP4-LISTEN:5500 TCP6:localhost:5500**

**Comment by hsanjuan** [comment]
> socat TCP4-LISTEN:5500 TCP6:localhost:5500

To me it says address already in use even though it is listening ipv6. Using 5001 and pointing the client to that does not work either, not sure why.
